src/__tests__/sessions-test.ts

```diff
@@ -0,0 +1,43 @@
+import {onSessionRenewed, renewSession} from '../sessions';
+import {
+    createFakeAndroidPostMessage,
+    removeFakeAndroidPostMessage,
+} from './fake-post-message';
+
+afterEach(() => {
+    removeFakeAndroidPostMessage();
+});
+
+test('webapp requests session renewal to native app', async () => {
+    const newAccessToken = 'any token';
+    createFakeAndroidPostMessage({
+        checkMessage: message => {
+            expect(message.type).toBe('RENEW_SESSION');
+        },
+        getResponse: message => ({
+            type: message.type,
+            id: message.id,
+            payload: {accessToken: newAccessToken},
+        }),
+    });
+
+    const accessToken = await renewSession();
+    expect(accessToken).toEqual(newAccessToken);
```

Comment:
I've made the message to return an object with the key `accessToken` for the same reason. To make future changes retro-compatibles at bridge level. I think it's important at that level because we always need to be backward compatible with old app versions.
But I think making the library itself retro-compatible it's not a goal per se, we can release major versions for breaking changes, not a big problem there. I also want to make the api as simple as posible and make both functions (`renewSession` and `onSessionRenewed`) symmetrical.
```js
onSessionRenewed(newAccessToken => {});
const newAccessToken = await renewSession();
``` 
I don't want one function to return an object and the other function to return an string.